mathesar_ui/src/sections/table-view/TableView.svelte

```diff
@@ -120,6 +122,13 @@
       void deleteRecords(database, identifier, selectedEntries);
     }
   }
+
+  function columnAdd(event) {
+    const newColumn:TableColumn = (event.detail) ? <TableColumn>event.detail : null;
```

Comment:
Please refrain from using the angle syntax for casting. While this is completely valid, as a convention, we intend to use the new 'as' keyword.

```suggestion
    const newColumn:TableColumn = (event.detail ?? null) as TableColumn;
```

Reply:
https://github.com/pegnott/mathesar/blob/754414d202a29033bd437f2d845269d5885a5831/mathesar_ui/src/sections/table-view/TableView.svelte#L127